.github/ISSUE_TEMPLATE/enhancements-review

```diff
@@ -0,0 +1,79 @@
+---
+name: Enhancement 🚀
+about: Track high-level project progress across multiple areas.
+---
+
+# **Summary**
+<!-- 
+A brief overview of what this enhancement aims to achieve and why it matters.
+Focus on the problem it solves, why it's important, and what outcome we want.
+-->
+_A brief overview._
+
+---
+
+## **Scope**
+
+### ✅ **In Scope**
+<!-- 
+Clearly define what's included in this enhancement.
+What work or features are part of this? Be specific.
+-->
+- 
+
+### ❌ **Out of Scope**
+<!-- 
+Define what is explicitly excluded to prevent scope creep.
+Mention anything that might sound related but won’t be tackled here.
+-->
+- 
+
+---
+
+## **Responsible Areas**
+<!-- 
+Pick one or more of the following areas this enhancement touches:
+Ironcore API, Metal Automation, Networking, Virtualization, Storage, UI, Gardener Extension, Documentation
+-->
+- 
+
+---
+
+
+## **Contributors**
+<!-- 
+List all contributors, collaborators, or teams actively participating in this enhancement.
+Include GitHub handles.
+-->
+- @exampleuser1  
+- @exampleuser2
+---
+
+## **Acceptance Criteria**
+<!-- 
+Organize by area (if needed), and list what needs to be true for this enhancement to be considered “done.”
+-->
+
+- ### Area 1
+  - [ ]
+
+- ### Area 2
+  - [ ]
+
+- ### Common
+  - [ ]
+
+---
+
+## **Action Items**
+<!-- 
+Please ensure all items below are checked before or during issue creation.
+Make sure sub-issues are created for each relevant area.
+Add the issue to the "Roadmap" project.
+-->
+- [ ] Assign labels (e.g., `area/ironcore-api`, `kind/design`)
```

Comment:
Can we harmonize the `area` labels? Instead of `area/ironcore-api` i would suggest to use `area/iaas` `area/baremetal`.

Reply:
Sounds good to me, I will adapt the labeling for both area/iaas, and extensions.